__tests__/app.test.js

```diff
@@ -90,3 +91,58 @@ describe("Err /api/articles/article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: responds with an articles array of article objects and a new comment_count property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((column) => {
+          expect(typeof column.author).toBe("string");
+          expect(typeof column.title).toBe("string");
+          expect(typeof column.article_id).toBe("number");
+          expect(typeof column.topic).toBe("string");
+          expect(typeof column.created_at).toBe("string");
+          expect(typeof column.votes).toBe("number");
+          expect(typeof column.article_img_url).toBe("string");
+          expect(typeof column.comment_count).toBe("string");
+        });
+      });
+  });
+  test("200: sorts the data by default by created_at in descending", () => {
+    return request(app)
+      .get(`/api/articles?sort_by=created_at`)
```

Comment:
The data should be sorted like this by default, with no query added. 
Queries are not implemented until task 11